aurora/bag_transfer/templates/transfers/footer.html

```diff
@@ -4,5 +4,6 @@
   	<img src="{% static "dist/img/RAC-logo.png" %}" alt="The Rockefeller Archive Center">
   </a>
   &copy; 2017 Rockefeller Archive Center. Aurora code available under the
-  <a href="https://github.com/RockefellerArchiveCenter/aurora/blob/master/LICENSE">MIT License.</a>
+  <a href="https://github.com/RockefellerArchiveCenter/aurora/blob/master/LICENSE">MIT License.</a> 
```

Comment:
```suggestion
  <a href="https://github.com/RockefellerArchiveCenter/aurora/blob/base/LICENSE">MIT License.</a> 
```

Reply:
I just see that we never updated this license link when we changed from master to base, so looks like a good time to update!